### DISCUSSION TITLE
Fortofolio

### DISCUSSION THREAD

**Discussion by Perlicuyy** [community/community#126660] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

<title>Portfolio Firly Okta Tri Ramadhan</title> <style> body { font-family: Arial, sans-serif; margin: 0; padding: 0; background-color: #87CEEB; } .container { max-width: 800px; margin: 0 auto; padding: 20px; background-color: #FFFFFF; box-shadow: 0 0 10px rgba(0, 0, 0, 0.1); } .header { text-align: center; } .header img { width: 150px; border-radius: 15%; } .header h1 { margin: 10px 0; font-size: 2em; } .header p { color: #000000; } .content { margin: 20px 0; } .content h2 { color: #333; } .content p { line-height: 2; } .content img { max-width: 100%; } </style>
title>Horizontal Rule</title>
README
GPL-3.0 license
<title>Portfolio Firly Okta Tri Ramadhan</title> <style> body { font-family: Arial, sans-serif; margin: 0; padding: 0; background-color: #87CEEB; } .container { max-width: 800px; margin: 0 auto; padding: 20px; background-color: #FFFFFF; box-shadow: 0 0 10px rgba(0, 0, 0, 0.1); } .header { text-align: center; } .header img { width: 150px; border-radius: 15%; } .header h1 { margin: 10px 0; font-size: 2em; } .header p { color: #000000; } .content { margin: 20px 0; } .content h2 { color: #333; } .content p { line-height: 2; } .content img { max-width: 100%; } </style>
Firly Okta Tri Ramadhan
Kota Serang , Banten | firlyramadhan410@gmail.com | 0895-1838-4349

<title>Horizontal Rule</title>
RINGKASAN
<title>Horizontal Rule</title>
Perkenalkan saya Firly Okta Tri Ramadhan, saya punya panggilan yang unik dari keluarga,teman,tetangga sekitar rumah saya yaitu (Madang) tapi faktanya teman di sekolah hanya tau panggilan saya Firly.Saat ini aktif sebagai siswa SMA 3 KOTA SERANG

PENGALAMAN KERJA & ORGANISASI
<title>Horizontal Rule</title>
Atletik (hanya di kelas 10)
PENDIDIKAN
<title>Interactive Button with Logo</title> <style> .button { display: inline-block; padding: 10px 20px; font-size: 16px; color: white; background-color: #007BFF; border: none; border-radius: 5px; text-align: center; text-decoration: none; cursor: pointer; } .button img { vertical-align: middle; margin-right: 10px; height: 24px; /* Adjust based on your needs */ } .button:hover { background-color: #0056b3; } </style> <title>Horizontal Rule</title>
SMAN 3 KOTA SERANG

<title>Horizontal Rule</title>
<title>Interactive Button with Logo</title> <style> .button { display: inline-block; padding: 10px 20px; font-size: 16px; color: white; background-color: #007BFF; border: none; border-radius: 5px; text-align: center; text-decoration: none; cursor: pointer; } .button img { vertical-align: middle; margin-right: 10px; height: 24px; /* Adjust based on your needs */ } .button:hover { background-color: #0056b3; } </style>
![Screenshot_20230811_163649_Gallery](https://github.com/community/community/assets/168792552/16603b60-66a7-4338-a037-7c21e5abcf11)

**Comment by Raunaksplanet** [comment]
Here is your fixed code

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Portfolio Firly Okta Tri Ramadhan</title>
    <style>
        body {
            font-family: Arial, sans-serif;
            margin: 0;
            padding: 0;
            background-color: #87CEEB;
        }

        .container {
            max-width: 800px;
            margin: 0 auto;
            padding: 20px;
            background-color: #FFFFFF;
            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
        }

        .header {
            text-align: center;
        }

        .header img {
            width: 150px;
            border-radius: 15%;
        }

        .header h1 {
            margin: 10px 0;
            font-size: 2em;
        }

        .header p {
            color: #000000;
        }

        .content {
            margin: 20px 0;
        }

        .content h2 {
            color: #333;
        }

        .content p {
            line-height: 2;
        }

        .content img {
            max-width: 100%;
        }

        .button {
            display: inline-block;
            padding: 10px 20px;
            font-size: 16px;
            color: white;
            background-color: #007BFF;
            border: none;
            border-radius: 5px;
            text-align: center;
            text-decoration: none;
            cursor: pointer;
        }

        .button img {
            vertical-align: middle;
            margin-right: 10px;
            height: 24px;
        }

        .button:hover {
            background-color: #0056b3;
        }
    </style>
</head>
<body>
    <div class="container">
        <div class="header">
            <img src="https://github.com/community/community/assets/168792552/16603b60-66a7-4338-a037-7c21e5abcf11" alt="Profile Picture">
            <h1>Firly Okta Tri Ramadhan</h1>
            <p>Kota Serang, Banten | firlyramadhan410@gmail.com | 0895-1838-4349</p>
        </div>
        <div class="content">
            <h2>RINGKASAN</h2>
            <p>Perkenalkan saya Firly Okta Tri Ramadhan, saya punya panggilan yang unik dari keluarga, teman, tetangga sekitar rumah saya yaitu (Madang) tapi faktanya teman di sekolah hanya tau panggilan saya Firly. Saat ini aktif sebagai siswa SMA 3 KOTA SERANG</p>

            <h2>PENGALAMAN KERJA & ORGANISASI</h2>
            <p>Atletik (hanya di kelas 10)</p>

            <h2>PENDIDIKAN</h2>
            <p>SMAN 3 KOTA SERANG</p>
        </div>
    </div>
</body>
</html>

```

**Comment by ebndev** [comment]
Hi @Perlicuyy! I hope @Raunaksplanet's answer solved your issue.

I will be closing this discussion as we try to keep the Community tidy and I don't see that you were asking a specific question. Please feel encouraged to post again in the future if you have things you'd like to share!
